tm2/pkg/db/common_test.go

```diff
@@ -1,65 +1,65 @@
-package db
+package db_test
 
 import (
-	"bytes"
 	"encoding/binary"
 	"fmt"
-	"math/rand"
 	"sync"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
+	"github.com/gnolang/gno/tm2/pkg/db"
+	"github.com/gnolang/gno/tm2/pkg/db/internal"
+	"github.com/jaekwon/testify/assert"
+	"github.com/jaekwon/testify/require"
 )
```

Comment:
It's probably out of scope of this PR, but we definitely need to get rid of one of these packages.